web/src/beta/features/PluginPlayground/Plugins/constants.ts

```diff
@@ -15,15 +15,20 @@ export type FileType = {
 export const REEARTH_YML_FILE = {
   id: "reearth-yml",
   title: "reearth.yml",
-  sourceCode: `
-        id: test-plugin
-        name: Test plugin
-        version: 1.0.0
-        extensions:
-            - id: test-widget
-                type: widget
-                name: Test
-    `,
+  sourceCode: `id: test-plugin
+name: Test plugin
+version: 1.0.0
+extensions:
+  - id: button
+    type: widget
+    name: Button
+    description: Button widget
```

Comment:
Here we are mentioning that the widget is has a relation or includes a button but looking at the source code, there was no button. Small issue for now but might be good to rename it so that it relates more to the source code. Maybe `Demo widget` ?

Reply:
fixed this: 

https://github.com/reearth/reearth-visualizer/pull/1224/commits/1bec19edc93e2cebc4f89b876c578439a63a394f

Reply:
You're right; it did feel a bit off... I’ve changed the name to "demo widget."

https://github.com/reearth/reearth-visualizer/pull/1224/commits/031c2ab0223d302da4f918194ec0f35829fff4e9